server/routes/makeAReferral/risk-information/oasys/arnRiskSummaryView.test.ts

```diff
@@ -0,0 +1,182 @@
+import riskSummaryFactory from '../../../../../testutils/factories/riskSummary'
+import supplementaryRiskInformationFactory from '../../../../../testutils/factories/supplementaryRiskInformation'
+import { Risk } from '../../../../models/assessRisksAndNeeds/riskSummary'
+import ArnRiskSummaryView from './arnRiskSummaryView'
+
+describe('ArnRiskSummaryView', () => {
+  describe('supplementaryRiskInformationArgs', () => {
+    describe('additionalRiskInformation', () => {
+      it('returns no information label if no additional risk information has been provided', () => {
+        const riskSummary = riskSummaryFactory.build()
+        const view = new ArnRiskSummaryView(
+          riskSummary,
+          supplementaryRiskInformationFactory.build({
+            riskSummaryComments: undefined,
+          })
+        )
+        expect(view.supplementaryRiskInformationArgs.additionalRiskInformation.label).toHaveProperty('text', 'None')
+      })
+    })
+
+    describe('summary', () => {
+      it('returns null text when undefined redactedRisk provided', () => {
+        const view = new ArnRiskSummaryView(
+          riskSummaryFactory.build(),
+          supplementaryRiskInformationFactory.build({ redactedRisk: undefined })
+        )
+        expect(view.supplementaryRiskInformationArgs.summary.whoIsAtRisk.text).toBeNull()
+        expect(view.supplementaryRiskInformationArgs.summary.natureOfRisk.text).toBeNull()
+        expect(view.supplementaryRiskInformationArgs.summary.riskImminence.text).toBeNull()
+        expect(view.supplementaryRiskInformationArgs.riskToSelf.suicide.text).toBeNull()
+        expect(view.supplementaryRiskInformationArgs.riskToSelf.selfHarm.text).toBeNull()
+        expect(view.supplementaryRiskInformationArgs.riskToSelf.hostelSetting.text).toBeNull()
+        expect(view.supplementaryRiskInformationArgs.riskToSelf.vulnerability.text).toBeNull()
+      })
+
+      it('returns correct text when supplementary information provided', () => {
```

Comment:
_not a blocker_
🙋‍♂️ would be useful to define what is "correct" in case this test regresses -- if I were trying to fix this regression, would I have enough knowledge about what is correct?